code/go/0chain.net/blobber/settings.go

```diff
@@ -120,27 +118,62 @@ func setMaxFileSizeFromChain() error {
 	}
 
 	config.StorageSCConfig.MaxFileSize = cb.mfs
+
 	return nil
 }
 
 func updateCCTWorker(ctx context.Context) {
-	ticker := time.NewTicker(time.Hour)
-
-	for {
-		select {
-		case <-ctx.Done():
-			return
-		case <-ticker.C:
-			// We'd panic if err occurred when calling setCCTFromChain from
-			// main.go file because cct would be initially 0 and we cannot
-			// work with 0 value.
-			// Upon updating cct, we only log error because cct is not 0
-			// We should try to submit challenge as soon as possible regardless
-			// of cct value.
-			err := setCCTFromChain()
-			if err != nil {
-				logging.Logger.Error(err.Error())
+	go func() {
```

Comment:
why do we need goroutine here?

Reply:
To run updateCctWorker  and maxFileSizeUpdateWorker  simultaneously.